### ISSUE TITLE
Do not load resources on checkout

### ISSUE THREAD

**Issue by SDavisMedia** [easydigitaldownloads/edd-free-downloads#183]
A customer is having issues with the EDD `Show Terms` link not working on checkout. The problem goes away when Free Downloads is deactivated.

I was able to replicate this quickly, without checking any of my plugin versions. But then I switched back to FD `master` (can't even remember what branch I was on) and pulled down. I can't replicate now, and I can't find where I was able to replicate again. So... I'm useless, but I can at least say I replicated once.

These errors show up on the customer's site: 
<img width="990" alt="screenshot 2017-09-05 16 05 06" src="https://user-images.githubusercontent.com/2359131/30087188-0fea7b9c-9254-11e7-83c7-68cf0ee45450.png">

They actually show on every page of the site, though, because I think FD's resources load on every page. That was done so `[purchase_link]` could be used on any page of the site. But I do wonder if it's necessary to load them on checkout. That would most like resolved the customer's issue.

I don't know if that's the right solution. I'm not sure what else to do, though. Here's a product on the customer's site: http://www.martialartsselfdefence.co.uk/downloads/jun-fan-gung-fu-jeet-kune-do-volume-three-dvd/ Add that to the cart and proceed to checkout to see the `Show Terms` link broken.

**Comment by dannyb195** [comment]
I was able to reliably reproduce this issue using the 2017 theme and up-to-date EDD and EDD Free Downloads. After reviewing the user's site I noticed that Thickbox ([https://wordpress.org/plugins/thickbox/](https://wordpress.org/plugins/thickbox/)) is installed.

My local installed worked as expected until I activated Thickbox. This said unchecking the "Include jQuery" box on the Thickbox settings page did fix the issue.
<img width="368" alt="screen shot 2017-09-21 at 1 28 23 pm" src="https://user-images.githubusercontent.com/247887/30712229-ec8f0e9a-9ed0-11e7-8196-31537a885580.png">
